app/controllers/task_controller.rb

```diff
@@ -30,12 +30,45 @@ def delete
   	redirect_to action: "index"
   end
 
+  def edit
```

Comment:
In line #10 above - You can just use `Task.find` directly using the selected_id, you don't need find_by with the specific column and string interpolation